### DISCUSSION TITLE
Find

### DISCUSSION THREAD

**Discussion by thiyashoksanju** [lxqt/lxqt#2365] [Q&A]
Where i can find the code of Active connections popup showed when left clicking/mouse pressing the network monitor on panel. I believe it is a dynamic widget created during run time and not by any design .ui file. Correct me if I am wrong and please lead me to the exact implementation place in source code.
1. Is the popup dynamically created or already designed by UI file
2. Where is the exact code of this popup located in lxqt panel or any other repository

**Comment by edwardp3** [comment]
I just added the Network Monitor applet to the LXQt Panel (LXQt 1.2.0, Debian unstable, amd64). When I left-click it, nothing occurs. Right-click / Configure Network Monitor, allows the Icon and Interface selections to be changed.

**Reply by thiyashoksanju** [reply]
Guys i am talking about the popup from attached image. I am doing a study of customizing the UI of this popup. This popup is shown when i do mouse press on network monitor applet. 
But i could not find the implementation part of this one. Any leads on this would be more helpful.
![network (1)](https://user-images.githubusercontent.com/22815534/211772719-7e22793d-25b5-46f2-8e69-062902e25b98.png)

**Reply by palinek** [reply]
> Guys i am talking about the popup from attached image.

It seems like you're using the [nm-tray](https://github.com/palinek/nm-tray)

**Reply by thiyashoksanju** [reply]
yes. Thanks @palinek for sharing. One more clarification. Does the password prompt of new connections also comes under nm-tray UI?
